.devcontainer/README.md

```diff
@@ -114,3 +114,80 @@ Click the badge above or [click here](https://codespaces.new/NVIDIA/cccl?quickst
 For more information, see the `.devcontainer/make_devcontainers.sh --help` message.
 
 **Note**: When adding or updating supported environments, modify `matrix.yaml` and then rerun this script to synchronize the `devcontainer` configurations.
+
+## Dev Containers on WSL
```

Comment:
I would love to move that to some sub document that focusses on windows

Reply:
Since that's in `README` do you have any other potential file name I could use within the same dir?